flink-formats/flink-avro-glue-schema-registry/src/main/java/org/apache/flink/formats/avro/glue/schema/registry/GlueSchemaRegistryOutputStreamSerializer.java

```diff
@@ -72,17 +72,17 @@ public void registerSchemaAndSerializeStream(Schema schema, OutputStream out, by
                 glueSchemaRegistrySerializationFacade.encode(
                         transportName,
                         new com.amazonaws.services.schemaregistry.common.Schema(
-                                schema.toString(), "Avro", getSchemaName()),
+                                schema.toString(), "AVRO", getSchemaName()),
```

Comment:
Is there not a constant? `DataFormat.AVRO` maybe?

Reply:
Yess I can change to use constant.